vendor/github.com/json-iterator/go/Gopkg.lock

```diff
@@ -0,0 +1,21 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
Do we need `Gopkg.lock` and `Gopkg.toml` files? I know these are generated using `dep ensure`. Either way is fine with me.